### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by ShingekiNoRex**
# SCP:SL Translation PR Template

## Language
english

## Type of change
- [ ] Added new language translation
- [x] Changed a currently existing translation

## Have you tested the changes you've made?
- [x] Yes
- [ ] No

# Checklist:
- [ ] I have made sure to not commit changes to other stuff than my own or the default english one
- [ ] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [x] I have made sure that I'm not submitting a duplicate translation 
- [x] I have not included any advertisements/pointers to non-game stuff other than author in the folder name

### CODE REVIEW

File: Translations/zh-Hans/MainMenu.txt [hunk 1]
```diff
@@ -80,4 +80,6 @@ SCP-079 - 后处理
 为临时血量建立独立模型，而不是直接覆盖于实血血条上
 我们的翻译已在GitHub开源！\n想要来帮忙吗？点击此处提交反馈。
 更多状态展示
-一旦开启，你的Discord和Steam将显示你在游戏内的具体数据。
+一旦开启，你的Discord和Steam将显示你在游戏内的具体数据
+显示具体血量
+显示具体血量数值，而不仅仅是百分比
```
Comment (SchrodingerSig) [inline Translations/zh-Hans/MainMenu.txt]: “不仅仅是"感觉有点怪怪的？
line 84感觉改成”显示**实际**血量“会好点

Comment (ShingekiNoRex) [reply Translations/zh-Hans/MainMenu.txt]: 那描述部分怎么改

Comment (SchrodingerSig) [reply Translations/zh-Hans/MainMenu.txt]: 显示实际血量，而非血量百分比
↑差不多这样？

Comment (SchrodingerSig) [reply Translations/zh-Hans/MainMenu.txt]: ”just the percentage“的just肯定不能翻”仅仅是“

**Pull request merged**